### DISCUSSION TITLE
Taro升级3.6.7后，编译es2020新语法报错

### DISCUSSION THREAD

**Discussion by codMeing** [NervJS/taro#13956] [Q & A]
taro升级到3.6.7后，需要在babel.config.js中手动添加以下plugins才可以成功编译？否则可选链式操作符（?.）和空值合并（??）编译报错，taro3.3.x没有该问题。
@babel/plugin-proposal-optional-chaining
@babel/plugin-proposal-nullish-coalescing-operator

**Comment by TheKonka** [comment]
![image](https://github.com/NervJS/taro/assets/22173084/b03018d9-2129-43ee-8616-de68998c3241)

什么报错呢？我也是最新版本默认配置，没有遇到报错

**Reply by yoyo837** [reply]
他意思是是不是说, 非Taro的部分, 也就是自己的源码部分有用到`?.`、`??`, 但Taro默认配置不支持呢?

**Reply by codMeing** [reply]
是这个意思，没升级前业务代码中`?.`，`??`不需要额外plugins配置

**Reply by yoyo837** [reply]
听上去是Taro新版本反而去掉了`?.`，`??`的默认支持.  帮挂个issue呗老哥

**Reply by TheKonka** [reply]
> 听上去是Taro新版本反而去掉了`?.`，`??`的默认支持. 帮挂个issue呗老哥

怎么可能去掉，我用都是正常的

**Reply by yoyo837** [reply]
@codMeing 来个复现

**Reply by codMeing** [reply]
找到原因了，因为业务仓库中写的browserslist未被 babel-loader 识别转换，使用taro-cli新建的项目或调整browserslist浏览器前缀值，使其能够被识别转换即可。

还未细看taro3.3.x到3.6.7这方面的改动
